cypress/utils/productsUtils.js

```diff
@@ -1,152 +1,130 @@
-import Attribute from "../apiRequests/Attribute";
-import Category from "../apiRequests/Category";
-import Product from "../apiRequests/Product";
+import * as attributeRequest from "../apiRequests/Attribute";
+import * as categoryRequest from "../apiRequests/Category";
+import * as productRequest from "../apiRequests/Product";
 
-class ProductsUtils {
-  productRequest = new Product();
-  attributeRequest = new Attribute();
-  categoryRequest = new Category();
+let product;
```

Comment:
Having those objects on module leevel seems error-prone. Better create function which returns product or other object and save it in test scope.